nodecore/api-calls-examples/nodecore-pending-transactions-webpage/src/api/utils.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable  @typescript-eslint/no-non-null-assertion */
+
+import { IJsonRPC } from '../models/IJsonRPC';
+
+export const request = async <T>(method: string, params = {}): Promise<T> => {
```

Comment:
I think you also need to handle code 500 (nodecore error)

Reply:
done